app/modules/auth/auth.factory.js

```diff
@@ -13,11 +13,11 @@ function authFactory($interval, $timeout, $location, $window, localStorageServic
     var _user = {};
 
     OAuth.checkCurrent()
-      .then(function (user) {
-        if (user) {
-            _setUserData();
-        }
-      });
+        .then(function (user) {
```

Comment:
do we want to keep these whitespace changes only?

Reply:
Yes please

Reply:
why is the indenting at 4 spaces?

Reply:
Because it was like that when I started working on this app and it was too late when I realised I could change it. It annoys me every time I work on it.

Reply:
ok - this PR has the change in whitespace here it seems unrelated and that's why i'm confused by it. It's not blocking for sure, just this PR is adding spacing to a function for some reason...